lib/ssl/src/dtls_connection.erl

```diff
@@ -448,9 +448,12 @@ init({call, From}, {start, Timeout},
     Hello = dtls_handshake:client_hello(Host, Port, ConnectionStates0, SslOpts,
 					Session#session.session_id, Renegotiation, Cert),
 
+    MaxFragEnum = maps:get(max_frag_enum, Hello#client_hello.extensions, undefined),
+    ConnectionStates1 = ssl_record:set_max_fragment_length(MaxFragEnum, ConnectionStates0),
```

Comment:
Can it be a problem if we allow the fragmentation of a ClientHello in DTLS? @IngelaAndin

Reply:
It should already be allowed, even if it may not happen!